python/cudf/cudf/_libxx/copying.pyx

```diff
@@ -419,9 +435,14 @@ def column_split(Column input_column, object splits):
     return result
 
 
-def table_split(Table input_table, object splits):
+def table_split(Table input_table, object splits, keep_index=True):
```

Comment:
```suggestion
def table_split(Table input_table, object splits, bool keep_index=True):
```
Just a minor change to be consistent with peer function in the same file.

Reply:
I always miss that bool